src/applications/mhv/secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -44,14 +44,18 @@ class PatientComposePage {
   };
 
   verifySendMessageConfirmationMessage = () => {
-    cy.get('.vads-u-margin-bottom--1').should(
+    cy.get('.main-content > va-alert').should(
       'have.text',
       'Secure message was successfully sent.',
     );
   };
 
   verifySendMessageConfirmationMessageHasFocus = () => {
-    cy.get('.vads-u-margin-bottom--1', { timeout: 5000 }).should('be.focused');
```

Comment:
@fazilqa @haniasma I would suggest creating a reference to an alert and then use it in other methods. that way if you need to change a locator - that would happen in one place